css/css-flexbox_navigation/css/styles.css

```diff
@@ -29,3 +29,19 @@ ul {
   padding: 5px 14px;
   border-radius: 5px;
 }
+
+.navigation__menu {
+  display: flex;
+  justify-items: center;
+  flex-direction: row;
```

Comment:
This is the default value. I don't think you need to specify it unless another style is setting this to another value than default.

Reply:
I mean `flex-direction: row;` only. I'm not sure why GitHub is highlighting the whole block 🤔 Sorry for the confusion!